src/libs/ReportUtils.ts

```diff
@@ -4618,6 +4625,61 @@ function shouldDisableThread(reportAction: OnyxEntry<ReportAction>, reportID: st
     );
 }
 
+function getAllAncestorReportActions(
+    report: Report | null | undefined,
+    shouldHideThreadDividerLine: boolean,
+    reports: OnyxCollection<Report> = {},
+    reportActions: OnyxCollection<ReportActions> = {},
+): Ancestor[] {
+    if (!report) {
```

Comment:
Hmm I think it would make sense to return early immediately if this is not a thread? Only threads could have ancestors?

Reply:
>Hmm I think it would make sense to return early immediately if this is not a thread?

Yeah, we can add that check. But right now the function is only used in `ReportActionItemParent` and it's already a check if it is a thread to render the component.

https://github.com/Expensify/App/blob/9d4c634a87890ccbdcc2efe353ae9d929aa5530b/src/pages/home/report/ReportActionsListItemRenderer.js#L53-L59

Reply:
Ok that makes sense I guess. If we ever use it more generally then I'd probably do an early return since this is a method that only works for "threads".